### ISSUE TITLE
Making Web Pages: Use your Personal Git Repo!

### ISSUE THREAD

**Issue by ghbondar** [ebeshero/DHClass-Hub#350]
BE A PRO! When creating files for web pages, be they for class, fun, or profit, remember to save your HTML, CSS, images, graphics, etc. files in your LOCAL personal Git repository that you've created for your personal webpage AND then transfer them to your personal web space accounts at newtfire.org using the FTP client. 
Finally, always add, commit, and push your files from your local Git repository to your MASTER personal GitHub repository to securely keep the official copies of your files on GitHub.

**Comment by ebeshero** [comment]
Also, when you need help with HTML and CSS, you can write up an issue here, and link out to your code on the web instance of your own GitHub repo. We can practice line-commenting over there and even fork/clone/download your code to tinker with there--and then share comments / highlights in code blocks on how to fix over here in the common Issues board, where we all can see!

**Comment by Jamielynn92** [comment]
How do I make tables go side by side instead of going underneath the first one?

**Comment by ebeshero** [comment]
Hi @Jamielynn92 ! There are a couple of ways to do this, but my favorite is with CSS flex properties. You want to patiently read and test a little bit, but I love this tutorial to explain how flexing works:

https://internetingishard.com/html-and-css/flexbox/

**Comment by ebeshero** [comment]
@Jamielynn92 That tutorial might be a lot to start out with, so if it's hard to follow, we'll try demonstrating some horizontal layouts with flex in class tomorrow! There are some HTML + CSS tricks it might help for us to explain first, especially CSS class selectors.

**Comment by Jamielynn92** [comment]
I looked it up and found one but I'm not sure how to css it. I used this for the first table <table style="display: inline-block; border: 1px solid; float: left; "> and this for the second <table style="display: inline-block; border: 1px solid; ">. It took out the lines between each cell though

**Comment by Jamielynn92** [comment]
also, I found the website I made it high school.

**Comment by Jamielynn92** [comment]
@ebeshero I'm not gunna lie, I'm so confused on how to make the CSS. Its been too long. I've looked it over, but it is confusing.

**Comment by ebeshero** [comment]
@Jamielynn92 It looks like you're coding your CSS styles directly inside your HTML as HTML attributes, which I imagine is how you did this in high school. It isn't quite the way we want you to be writing CSS here--we want you to learn a different way! We want you to write a set of CSS rules in a separate file, which you can connect with a `<link>` element in your HTML file. This is different from how you used to write this code, but please bear with us and try it our way--there is good reason for it! 

Why write a separate CSS file and link it to your HTML? Because you are going to make websites that gave multiple pages that share the same style rules. You only have to write those rules *once* in a single separate file and then link them to as many HTML pages as you like--that saves a LOT of work later and most importantly makes your HTML simpler to read. Please follow how we are guiding you in our tutorial pages for [HTML](http://dh.newtfire.org/explainHTML.html) and [CSS](http://dh.newtfire.org/explainCSS.html).

**Comment by Jamielynn92** [comment]
Which file am I putting the <link on.

**Comment by ebeshero** [comment]
@Jamielynn92 You're setting it in the HTML element inside its `<head>`, just like we show you in the tutorial. You can just copy and paste the line into your file and just make sure it's pointing to your CSS file instead of our made-up filename.

**Comment by Jamielynn92** [comment]
So my CSS should dictate my HTML. So I shouldn't put any extra tags inside my html? Cause I've been going through different ways for my CSS, and nothing is working for me. its frustrating me a lil. and most of the examples have it inside the HTML as well. I'm trying to grasp it, but separating them is whats getting me.

**Comment by ebeshero** [comment]
@Jamielynn92 Our examples don't show CSS inside HTML, and we really try to make that a standard practice--you can see some examples in student projects on newtfire. We just want to minimize the CSS you put inside HTML tags b/c it contributes to "code bloat"--it makes HTML hard to read. Save your CSS in a separate file.

Here's an example you can look at by [viewing the page source](view-source:http://hamilton.newtfire.org/) on the main page of the Hamilton Project from last spring.

**Comment by ebeshero** [comment]
view-source:http://hamilton.newtfire.org/

**Comment by ebeshero** [comment]
Find the view-page source option in your web browser as you're looking at that site and you can see its HTML code and open up its CSS: http://hamilton.newtfire.org/style.css

**Comment by Jamielynn92** [comment]
I may need to have a lil help in the morning with my image. I put one on, but I'm not sure how to align it with my text under the favorite tv shows heading.
